From 47d7bf74975958c823990f88f2dd06bed3a60765 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sun, 11 Sep 2016 11:06:34 +0100
Subject: [PATCH 01/12] applets/icon-tasklist: Add workaround for Pale Moon
 pinning

Pale Moon, for some unknown reason, decided to use a space in the WM
Class Name, leading to breakage in identification. Per issue #575 we
account for this in the simpletons register.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 panel/applets/icon-tasklist/AppSystem.vala | 1 +
 1 file changed, 1 insertion(+)

diff --git a/panel/applets/icon-tasklist/AppSystem.vala b/panel/applets/icon-tasklist/AppSystem.vala
index 527fef3..161a7e8 100644
--- a/panel/applets/icon-tasklist/AppSystem.vala
+++ b/panel/applets/icon-tasklist/AppSystem.vala
@@ -35,6 +35,7 @@ public class AppSystem : GLib.Object
             "fbreader",
             "google-chrome",
             "hexchat",
+            "pale moon",
             "spotify",
             "steam",
             "telegram",
-- 
2.10.0

